web/components/issues/peek-overview/root.tsx

```diff
@@ -93,42 +92,47 @@ export const IssuePeekOverview: FC<IIssuePeekOverview> = observer((props) => {
         data: Partial<TIssue>,
         showToast: boolean = true
       ) => {
-        try {
-          await updateIssue(workspaceSlug, projectId, issueId, data);
-          if (showToast)
-            setToastAlert({
+        const updateIssuePromise = updateIssue(workspaceSlug, projectId, issueId, data);
+        if (showToast) {
+          setPromiseToast(updateIssuePromise, {
+            loading: "Updating issue...",
+            success: {
               title: "Issue updated successfully",
-              type: "success",
-              message: "Issue updated successfully",
-            });
-          captureIssueEvent({
-            eventName: ISSUE_UPDATED,
-            payload: { ...data, issueId, state: "SUCCESS", element: "Issue peek-overview" },
-            updates: {
-              changed_property: Object.keys(data).join(","),
-              change_details: Object.values(data).join(","),
+              message: () => "Issue updated successfully",
+            },
+            error: {
+              title: "Issue update failed",
+              message: () => "Issue update failed",
             },
-            path: router.asPath,
-          });
-        } catch (error) {
-          captureIssueEvent({
-            eventName: ISSUE_UPDATED,
-            payload: { state: "FAILED", element: "Issue peek-overview" },
-            path: router.asPath,
-          });
-          setToastAlert({
-            title: "Issue update failed",
-            type: "error",
-            message: "Issue update failed",
           });
         }
+
+        await updateIssuePromise
```

Comment:
Same here, updating an issue should not trigger toast i think, please check with Dilip